backend/backends/graphite/graphite.go

```diff
@@ -85,6 +85,10 @@ type client struct {
 	legacyNamespace  bool
 }
 
+func (client *client) Run(ctx context.Context) error {
+	return client.sender.Run(ctx)
+}
```

Comment:
I've added this method to statsd backend but forgot to add it here. Sender wasn't sending :-\